src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -51,6 +51,13 @@ internal class FeatureManagementKeyValueAdapter : IKeyValueAdapter
                             keyValues.Add(new KeyValuePair<string, string>($"{FeatureManagementConstants.SectionName}:{featureFlag.Id}:{FeatureManagementConstants.EnabledFor}:{i}:Parameters:{kvp.Key}", kvp.Value));
                         }
                     }
+
+                    //
+                    // process RequirementType only when filters are not empty
+                    if (featureFlag.Conditions.RequirementType != null)
+                    {
+                        keyValues.Add(new KeyValuePair<string, string>($"{FeatureManagementConstants.SectionName}:{featureFlag.Id}:{FeatureManagementConstants.RequirementType}", featureFlag.Conditions.RequirementType));
```

Comment:
```suggestion
                        keyValues.Add(new KeyValuePair<string, string>(
                            $"{FeatureManagementConstants.SectionName}:{featureFlag.Id}:{FeatureManagementConstants.RequirementType}", 
                            featureFlag.Conditions.RequirementType));
```

Reply:
nit: I prefer this spacing to make it easier to read ^. Might have to also do it to the other large lines above.

Reply:
Thanks, committed as suggested.

> Might have to also do it to the other large lines above.

For other lines unrelated to this feature, I would prefer to create a separate PR for such style/format changes, against main branch.